### DISCUSSION TITLE
Multi target problem

### DISCUSSION THREAD

**Discussion by samanzamani** [livewire/livewire#5627] [1. Help] [answered]
Hi
I have a product table cart that includes products and quantity rows. Users can increase and decrease the quantity with the +,- button or it can change through an input value like the below image.
![Screenshot 2023-03-24 at 6 22 17 PM](https://user-images.githubusercontent.com/24776416/227559491-ce9fad9e-8e3f-44c4-8be5-1d3a35091177.png)

I have a `$peoducts` array which stores product items and three functions for increase, decrease, and remove items from the cart.
Now I want to change the `tr` tag opacity to 50 percent every time one of those functions is called.

Blade file: 
```blade
  @foreach($products as $index => $product)
  <tr wire:target="incrementQuantity({{ $index }}),decrementQuantity(({{ $index }}),products.{{ $index }},removeProduct({{$index}})"
  wire:loading.class="opacity-50">
....
```

Error:
![Screenshot 2023-03-24 at 6 27 09 PM](https://user-images.githubusercontent.com/24776416/227561178-c4126ae0-ad2d-42fa-91fd-0bffa39c9804.png)

If I use just function with the model it works:
```blade
  @foreach($products as $index => $product)
  <tr wire:target="incrementQuantity({{ $index }}),products.{{ $index }}"
  wire:loading.class="opacity-50">
....
```
OR
```blade
  @foreach($products as $index => $product)
  <tr wire:target="decrementQuantity({{ $index }}),products.{{ $index }}"
  wire:loading.class="opacity-50">
....
```

**Comment by joshhanley** [comment, marked as answer]
@samanzamani this is a known issue, please #4441 for more details 🙂
